hack/install.sh

```diff
@@ -61,53 +61,19 @@ install_required_packages() {
   fi
 }
 
+install_arkade(){
+  curl -sLS https://get.arkade.dev | sudo sh
+  arkade --help
+}
+
 install_cni_plugins() {
   cni_version=v0.9.1
-  suffix=""
-  arch=$(uname -m)
-  case $arch in
-  x86_64 | amd64)
-    suffix=amd64
-    ;;
-  aarch64)
-    suffix=arm64
-    ;;
-  arm*)
-    suffix=arm
-    ;;
-  *)
-    fatal "Unsupported architecture $arch"
-    ;;
-  esac
-
-  $SUDO mkdir -p /opt/cni/bin
-  curl -sSL https://github.com/containernetworking/plugins/releases/download/${cni_version}/cni-plugins-linux-${suffix}-${cni_version}.tgz | $SUDO tar -xvz -C /opt/cni/bin
+  sudo arkade system install cni --version ${cni_version} --path /opt/cni/bin
 }
 
 install_containerd() {
-  arch=$(uname -m)
-  CONTAINERD_VER=1.6.2
-  case $arch in
-  x86_64 | amd64)
-    curl -sLSf https://github.com/containerd/containerd/releases/download/v${CONTAINERD_VER}/containerd-${CONTAINERD_VER}-linux-amd64.tar.gz | $SUDO tar -xvz --strip-components=1 -C /usr/local/bin/
-    ;;
-  armv7l)
-    curl -sSL https://github.com/alexellis/containerd-arm/releases/download/v${CONTAINERD_VER}/containerd-${CONTAINERD_VER}-linux-armhf.tar.gz | $SUDO tar -xvz --strip-components=1 -C /usr/local/bin/
-    ;;
-  aarch64)
-      curl -sLSf https://github.com/containerd/containerd/releases/download/v${CONTAINERD_VER}/containerd-${CONTAINERD_VER}-linux-arm64.tar.gz | $SUDO tar -xvz --strip-components=1 -C /usr/local/bin/
-
-    ;;
-  *)
-    fatal "Unsupported architecture $arch"
-    ;;
-  esac
-
-  $SUDO systemctl unmask containerd || :
-  $SUDO curl -SLfs https://raw.githubusercontent.com/containerd/containerd/v${CONTAINERD_VER}/containerd.service --output /etc/systemd/system/containerd.service
-  $SUDO systemctl enable containerd
-  $SUDO systemctl start containerd
```

Comment:
No We don't need that. In local test on multipass containerd start after installation.

Reply:
Please just add the `--systemd` flag to make it explicit then when people come back to do a diff later if there's ever a problem, it will help understand the change.